ci/README.md

```
@@ -103,7 +109,7 @@ gsutil mb -l europe-west4  gs://bosh-gce-light-stemcell-ci-terraform-state
 ```
 
 Make buckets publicly readable
-```
+```shell
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)